docs/setup/setup.md

```diff
@@ -29,7 +29,10 @@ KubeEdge is composed  of cloud and edge parts. It is built upon Kubernetes and p
 
 + [Install mosquitto](https://mosquitto.org/download/)
 
-**Note:** Do not install **kubelet** and **kube-proxy** on edge side
+**Note:** 
++ Do not install **kubelet** and **kube-proxy** on edge side
++ If you use kubeadm to install kubernetes, the `Kubeadm init` command can not be followed by the "--experimental-upload-certs" or 
+"--upload-certs" flag
```

Comment:
Thanks for the quick fix, and what is the effect of these two flags on kubeedge?

Reply:
the pod can't be Ready on the edge node,you can see the details at :https://github.com/kubeedge/kubeedge/issues/1447

Reply:
I think it may not be related to this two flag..

Reply:
> I think it may not be related to this two flags..

Before that, I couldn't find any solution, but without these two symbols, I can solve the problem successfully. If they are not related, what causes the problem？Do you have any findings？

Forgive my English by the way.